### ISSUE TITLE
calls from -e are starting before register

### ISSUE THREAD

**Issue by joaquinjsb** [baresip/baresip#2919]
Hello,
When I do a call from -e command, the calls starts immediately without waiting for registration.

**Comment by cspiel1** [comment]
Not sure if this is an issue. I don't think that there is a wait for registration condition for the `-e` commands.

**Comment by maximilianfridrich** [comment]
It sounds like you are using BareSIP in ways where simply calling it from the command line with some option is not enough. You might have to develop a small application that listens to BareSIP events and then triggers certain events. That's what the multiple BareSIP interfaces have been developed for.

**Comment by joaquinjsb** [comment]
yes, I guess I should use the stdin at this point, or try the D-Bus

**Comment by joaquinjsb** [comment]
sorry for one more, and I hope last ping / tag,

@maximilianfridrich do you guys any index / file I can use for reference for the list of "events" that the baresip fires? so I can try to catch them, for example, when registration is successful ?

**Comment by maximilianfridrich** [comment]
Yes, see https://github.com/baresip/baresip/blob/main/include/baresip.h#L825-L862.

**Comment by joaquinjsb** [comment]
I started working on it, I just found the https://github.com/baresip/baresip/blob/main/modules/ctrl_tcp/ctrl_tcp.c,
but I'm not receiving events in "real time", even sending a message, I'm getting no response from the server on port 4444, while using the 5555 standard, it's properly sending the std output of the process, is it working properly as of now?
